Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -403,16 +420,30 @@ public override void Exit()
 
 	void SetupStateMethodMap()
 	{
-		_actor.physics.RegisterState( PhysicsStateType.Jumping,  new Jumping( _actor ) );
-		_actor.physics.RegisterState( PhysicsStateType.Falling,  new Falling( _actor ) );
-		_actor.physics.RegisterState( PhysicsStateType.Grounded, new Grounded( _actor ) );
-		_actor.physics.RegisterState( PhysicsStateType.Climbing, new Climbing( _actor ) );
-		_actor.physics.RegisterState( PhysicsStateType.Gliding,  new Gliding( _actor ) );
-		_actor.physics.RegisterState( PhysicsStateType.Sprinting, new Sprinting( _actor ) );	
+		_actor.physics.RegisterState( PhysicsStateType.Jumping,   new Jumping( _actor ) );
+		_actor.physics.RegisterState( PhysicsStateType.Falling,   new Falling( _actor ) );
+		_actor.physics.RegisterState( PhysicsStateType.Grounded,  new Grounded( _actor ) );
+		_actor.physics.RegisterState( PhysicsStateType.Climbing,  new Climbing( _actor ) );
+		_actor.physics.RegisterState( PhysicsStateType.Gliding,   new Gliding( _actor ) );
+		_actor.physics.RegisterState( PhysicsStateType.Sprinting, new Sprinting( _actor ) );
 	}
 	#endregion
 
-	/**
+	public bool InteractCheck( RaycastHit hitInfo )
+	{
+		Interactable interactable = hitInfo.collider.gameObject.GetComponent<Interactable>();

```

Comment:
I don't think that's a good idea. A better approach would be to unify cutting and clicking on buddies into a single "interactible" system. I'll add that to the GitHub issues.